Dockerfile

```
@@ -0,0 +1,30 @@
+FROM ubuntu
+
+LABEL version="0.1"
+LABEL description="A super docker image."
+
+RUN apt-get update
+
+RUN apt-get -y install curl git gnupg
+RUN curl -sL https://deb.nodesource.com/setup_12.x -O
+RUN bash setup_12.x
+
+RUN apt-get -y install nodejs
+
+RUN mkdir -p /home/app 
+
+WORKDIR /home/app
```

The `RUN` instruction here isn't useful, the `WORKDIR` instruction create the directories. You can find more about the instruction [here](https://docs.docker.com/reference/dockerfile/#workdir).

As a personal preference, I tend to specify my working directory as early as possible in a Dockerfile for readability.

```suggestion
WORKDIR /home/app
```